configs/dsdl/cityscapes.py

```diff
@@ -0,0 +1,70 @@
+_base_ = [
+    '../_base_/models/deeplabv3_r50-d8.py', '../_base_/default_runtime.py',
+    '../_base_/schedules/schedule_40k.py'
+]
+
+crop_size = (512, 1024)
+data_preprocessor = dict(size=crop_size)
+model = dict(data_preprocessor=data_preprocessor)
+# dataset settings
+dataset_type = 'DSDLSegDataset'
+data_root = 'data/CityScapes'
+img_prefix = 'original'
+train_ann = 'dsdl/set-train/train.yaml'
+val_ann = 'dsdl/set-val/val.yaml'
```

Comment:
Should we create the `dsdl` folder at mmsegmentation root path or at the datasets root path?

Reply:
this path will concate with root_path, so the actually path is "data/cityscapes/dsdl/set-train/train.yaml".

furthermore,  here is a typical example of dsdl dataset's file struct :
```
data
└── VOC12-seg
    ├── dsdl
    └── original
```